jenkins/ci.suse.de/cloud-mkphyscloud-qa-rally-test.yaml

```diff
@@ -0,0 +1,101 @@
+    publishers:
+      - mkphyscloud-qa-common-publishers
+
+    concurrent: true
+
+    parameters:
+      - string:
+          name: hw_number
+          default: "2"
+          description: Mandatory, number of the QA cloud server
+
+      - string:
+          name: image_name
+          default: jeos-rally
+          description: | 
+            Pre condition to have jeos image uploaded before running rally tests
+
+      - string:
+          name: scenario_name
+          description: Optional; scenario name which typically is an integer with a single letter
+
+      - string:
+          name: scenario_job_name
+          description: Optional; name of the scenario jenkins job that is used to trigger this job
+
+      - string:
+          name: scenario_build_number
+          description: Optional; scenario build number that triggered this job
+
+      - string:
+          name: rally_server
+          default: backup.cloudadm.qa.suse.de
+          description: Server hostname where rally is setup
+
+    builders:
+      - shell: |
+          #!/bin/bash
+          admin=crowbar$hw_number;
+          cloud=qa$hw_number;
+
+          # Create artifacts dir
+          export artifacts_dir=$WORKSPACE/.artifacts
+          rm -rf $artifacts_dir
+          mkdir -p $artifacts_dir
+          touch $artifacts_dir/.ignore
+
+          # Connect to rally server and run tests
+          ssh -T root@$rally_server "bash -s $cloud $image_name" <<'EOF'
+
+          cloud=$1
+          image_name=$2
+
+          source .openrc
+          # Check if jeos image already exist on cloud
+          openstack image show -c name --format value $image_name
+          image_exist=$?
+          # Upload jeos image
+          if [[ $image_exist != 0 ]]; then
+              curl -s http://download.suse.de/ibs/SUSE:/SLE-12-SP2:/Update:/JeOS/images/SLES12-SP2-JeOS-for-OpenStack-Cloud.x86_64-1.2.0-Build13.308.qcow2| \
+              openstack image create --public --disk-format qcow2 $image_name
+          fi
+          # Check if specific cloud deployemnt already exist
+          rally deployment list | grep $cloud
+          deployment_exist=$?
+          if [[ $deployment_exist != 0 ]]; then
+              # Creat new cloud deployment env
+              rally deployment create --filename $cloud-deployment.json --name $cloud
+          else
+              # Use existing deployment
+              rally deployment use $cloud
+          fi
+          # Run rally test
+          rally task start rally/samples/tasks/scenarios/nova/nova.json
+          # Generate run results output
+          out_dir=/root/results
+          rm -rf $out_dir
+          mkdir -p /root/results
+          rally task results > $out_dir/output.json
+          rally task report $out_dir/output.json --out $out_dir/output.html
+          EOF
+          ret=$?
+          # Copy test run reslust output to jenkins artifacts
+          scp root@$rally_server:/root/results/* $artifacts_dir
```

Comment:
Can you please store the results somewhere else additionally?
For example in some "/root/rally-results-archive" folder on backup?
Because currently only some number (10?) of artivacts_dirs will be saved.
This is needed for the future long term comparison. e.g cloud7 to cloud8.